src/main/scala/org/constellation/util/ServeUI.scala

```diff
@@ -55,18 +55,19 @@ object MyStandalone extends StyleSheet.Standalone {
   import dsl._
 
   "a" - (
-    color := Colors.intBlue
+    color :=! Colors.intBlue
   )
 
 }
 
 object MyStyles extends StyleSheet.Inline {
   import dsl._
+  import Colors._
 
   val hoverDark: StyleA = style("hover-dark")(
-    backgroundColor := Colors.ansiGrey,
+    backgroundColor(Color(ansiGrey)),
```

Comment:
heads up -- made these changes because using ```:=``` gives a warning. most of the time I can replace it with the typed action, except for "a" one above, where I used ```:=!``` to assign without the warning.